src/main/java/Model/Web/Specific/ProjectCreation.java

```diff
@@ -0,0 +1,44 @@
+package Model.Web.Specific;
+
+import Model.Web.PrettyObject;
+import com.google.gson.annotations.Expose;
+
+import java.util.List;
+
+public class ProjectCreation extends PrettyObject {
+
+    // PARAMETERS
+    @Expose
+    private String project_name;
+
+    @Expose
+    private String required_email;
```

Comment:
vidim ze ja by som toto riesil koncepcne cez non-empty list, ak teda nemas extra presvedcivy argument preco nie

Reply:
vsetko som implementoval z tvojho review, ale toto si nemyslim, ze je dolezite riesit, kedze to funguje a nikoho to neovplyvni a treba sa posunut. (sam nestiham ine veci a vkuse len timak)